Updating QS guides documentation (#8486)
fishtown-analytics/docs.getdbt.com: website/snippets/tutorial-document-your-models.md

    </File>

<Tabs>

<TabItem value="View in Catalog" >

[<Constant name="explorer" />](/docs/explore/explore-projects) provides powerful tools to interact with your dbt projects, including documentation:
1. From the IDE, run one of the following commands:
   - `dbt docs generate` if you're on <Constant name="core" />
   - `dbt build` if you're on the <Constant name="fusion_engine" />
2. Click **Catalog** in the navigation menu to launch <Constant name="explorer" />.

> 3. In the <Constant name="explorer" /> pane, click the dropdown menu next to your account name and change it from **Production** to **Development**.

Reviewer: "dbt" does not look like the account name, suggesting this change instead:
```suggestion
3. In the <Constant name="explorer" /> pane, click the dropdown menu where you can select the environment and change it from **Production** to **Development**.
```

Author: Awaiting word on where this actually comes from (if it's the same for everyone), but think we should still give a location to orient them as the menu isn't the most well-lit

> 3. In the <Constant name="explorer" /> pane, click the dropdown menu next to your account name and change it from **Production** to **Development**.

Author: ```suggestion
3. In the <Constant name="explorer" /> pane, click the environment selection dropdown menu at the top of the file tree and change it from **Production** to **Development**.
```

  <Lightbox src="/img/docs/collaborate/dbt-explorer/catalog-nav-dropdown.png" width="85%" title="View your development environment information." />

> 4. Select your project from the project dropdown.

Reviewer: ```suggestion
4. Select your project from the file tree.
```

5. Use the search bar or browse the resource list to find the `customers` model.
6. Click the model to view its details, including the descriptions you added.

  <Lightbox src="/img/docs/collaborate/dbt-explorer/example-model-details.png" width="85%" title="View your model's documentation and lineage in Catalog." />

<Constant name="explorer" /> displays your model's description, column documentation, data tests, and lineage graph. You can also see which columns are missing documentation and track test coverage across your project.

</TabItem>

<TabItem value="View in Studio IDE" >

Alternatively, you can view docs directly from the IDE. Keep in mind that this is a legacy view and doesn't offer the same level of interactivity as <Constant name="explorer" />.

1. In the IDE, run one of the following commands:
   - `dbt docs generate` if you're on <Constant name="core" />
   - `dbt build` if you're on the <Constant name="fusion_engine" />

> 2. From the navigation bar, click the **documentation icon** located to the right of the **branch name**.

Reviewer: When I hover over this icon, it says "View docs". Should we call it that instead for consistency with the UI?
```suggestion
2. From the navigation bar, click the **View docs** icon located to the right of the **branch name**.
```

>    <Lightbox src="/img/docs/collaborate/dbt-explorer/docs-icon.png" width="30%" title="The docs icon in the Studio IDE." />

Reviewer: ```suggestion
   <Lightbox src="/img/docs/collaborate/dbt-explorer/docs-icon.png" width="30%" title="The View docs icon in the Studio IDE." />
```

3. From **Projects**, select your project name and expand the folders.

> 4. Click **models** > **Marts** > **customers**.

Reviewer: ```suggestion
4. Click **models** > **marts** > **customers**.
```
